src/interfaces/stepper-interfaces/YourCompany/address-interface.ts

```diff
@@ -1,5 +1,6 @@
 /** Interface to define a base address. */
 export interface AddressIF {
+  id?: number
```

Comment:
I am aware of multiple definitions of AddressIF (eg, also in Create UI, shared components, and probably Filings UI).

It would be _great_ to have a single definition. Thoughts?

Reply:
Agreed!
Future: create shared AddressIF's?

Reply:
There is one already!

https://github.com/bcgov/bcrs-shared-components/blob/main/src/interfaces/addresses-interface.ts

Every project has their own 🙄